docs/src/main/sphinx/release/release-446.md

```diff
@@ -0,0 +1,69 @@
+# Release 446 (1 May 2024)
+
+## General
+
+* Improve performance of `INSERT` statements into partitioned tables when the
+  `retry_policy` configuration property is set to `TASK`. ({issue}`21661 `)
+* Improve performance of queries with complex grouping operations.  ({issue}`21726`)
+* Reduce delay before killing queries when the cluster runs out of memory. ({issue}`21719`)
+* Prevent assigning null values to non-null columns as part of a `MERGE`
+  statement. ({issue}`21619`)
+* Fix `CREATE CATALOG` statements including quotes in catalog names. ({issue}`21399`)
+* Fix potential query failure when a column name ends with a `:`. ({issue}`21676`)
+* Fix potential query failure when a [SQL routine](/routines) contains a label
+  reference in a `LEAVE`, `ITERATE`, `REPEAT`, or `WHILE` statement. ({issue}`21682`)
+* Fix query failure when [SQL routines](/routines) use the `NULLIF`, `IN`,
```

Comment:
"IN" and "SWITCH" are not user-visible functions. IN is a SQL construct (`x IN (...)`), and SWITCH is used to implement CASE expressions.